ui/components/DashboardComponent.js

```diff
@@ -723,6 +725,22 @@ class DashboardComponent extends React.Component {
       },
     }
   })
+  getDarkMuiTheme = () => createTheme({
+    shadows : ["none"],
+    palette : {
+      type : "dark",
+    },
+    overrides : {
+      MuiPaper : { root : { backgroundColor : '#363636' } },
+      MuiFormLabel : {
+        root : {
+          "&$focused" : {
+            color : "#00B39F",
+          },
+        }
+      },
+    }
+  })
```

Comment:
Would you extend on this?

Reply:
same for https://github.com/meshery/meshery/blob/ebd105eb5789527616d5811b3a70e4b19c71b41e/ui/components/DashboardComponent.js#L682 where it has been used by someone else. Is it required to be here? If yes, then why do we have global theme.

Reply:
Yes, It is required, Actually global theme is controlling the theme of that dashboard datatable, It was getting colors defined in the global theme, that's why I used this.

Reply:
In that case, please put the functions in a new file with the context/comment of where and why it is used. Add it in this folder: https://github.com/meshery/meshery/tree/master/ui/themes